yt_dlp/extractor/mave.py

```diff
@@ -0,0 +1,84 @@
+import json
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    get_element_by_id,
+)
+
+
+class MaveIE(InfoExtractor):
+    _VALID_URL = r'https?://(?P<channel_id>[a-z]+)\.mave\.digital/ep-(?P<short_id>[0-9]+)'
+    _TESTS = [
+        {
+            'url': 'https://ochenlichnoe.mave.digital/ep-25',
+            'md5': 'aa3e513ef588b4366df1520657cbc10c',
+            'info_dict': {
+                'id': 'ochenlichnoe-25',
+                'title': 'Между мной и миром: психология самооценки',
+                'description': 'md5:83183d7002dc32fbebc3ccecd4a1ac03',
+                'thumbnail': r're:https?://.*\.jpg$',
+                'ext': 'mp3',
+                'channel': 'Очень личное',
+                'channel_id': 'ochenlichnoe',
+                'channel_url': 'https://ochenlichnoe.mave.digital/',
+            },
+        },
+        {
+            'url': 'https://budem.mave.digital/ep-12',
+            'md5': 'e1ce2780fcdb6f17821aa3ca3e8c919f',
+            'info_dict': {
+                'id': 'budem-12',
+                'title': 'Екатерина Михайлова: "Горе от ума" не про женщин написана',
+                'description': 'md5:d9ce1fc1fb5fc7b7a4e7a0b84a7861c3',
+                'thumbnail': r're:https?://.*\.jpg$',
+                'ext': 'mp3',
+                'channel': 'Все там будем',
+                'channel_id': 'budem',
+                'channel_url': 'https://budem.mave.digital/',
+            },
+        },
+    ]
+
+    def _real_extract(self, url):
+        channel_id, short_id = self._match_valid_url(url).group('channel_id', 'short_id')
+
+        channel_url = f'https://{channel_id}.mave.digital/'
+
+        video_id = f'{channel_id}-{short_id}'
+
+        webpage = self._download_webpage(url, video_id)
+
+        # Format: "TITLE — Подкаст «CHANNEL»"
+        page_title = self._html_search_regex(r'<title>(.+?)</title>', webpage, 'title')
+        match = re.search(r'^(.+?)\s*—\s*(.+?)«(.+?)»', page_title)
+        title = match.group(1).strip()
+        channel = match.group(3).strip()
```

Comment:
Thanks to 51887484e46ab6015c041cb1ab626a55f25a03bd, we are now able to easily parse the JSON embedded in these webpages with `_search_nuxt_json`. I've gone ahead and merged master into your PR branch, so we can do this:

```suggestion
        channel_id, slug = self._match_valid_url(url).group('channel', 'id')
        display_id = f'{channel_id}-{slug}'
        webpage = self._download_webpage(url, display_id)
        data = traverse_obj(
            self._search_nuxt_json(webpage, display_id),
            ('data', lambda _, v: v['activeEpisodeData'], any, {require('podcast data')}))
```

(you'll need to import `require` and `traverse_obj` from `..utils.traversal`)